src/test/java/core/basesyntax/strategy/StrategyImplTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.handler.BalanceHandlerImpl;
+import core.basesyntax.strategy.handler.PurchaseHandlerImpl;
+import core.basesyntax.strategy.handler.ReturnHandlerImpl;
+import core.basesyntax.strategy.handler.SupplyHandlerImpl;
+import core.basesyntax.strategy.handler.TransactionHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StrategyImplTest {
+    private static Map<FruitTransaction.Operation, TransactionHandler> transactionHandlerMap;
+    private static StorageDao storageDao;
+    private static Strategy strategy;
+
+    @BeforeClass
+    public static void beforeClass() {
+        storageDao = new StorageDaoImpl();
+        transactionHandlerMap = new HashMap<>();
+        transactionHandlerMap.put(FruitTransaction.Operation.BALANCE,
+                new BalanceHandlerImpl(storageDao));
+        transactionHandlerMap.put(FruitTransaction.Operation.PURCHASE,
+                new PurchaseHandlerImpl(storageDao));
+        transactionHandlerMap.put(FruitTransaction.Operation.SUPPLY,
+                new SupplyHandlerImpl(storageDao));
+        transactionHandlerMap.put(FruitTransaction.Operation.RETURN,
+                new ReturnHandlerImpl(storageDao));
+        strategy = new StrategyImpl(transactionHandlerMap);
+    }
+
+    @Test
+    public void get_balanceTransactionHandler_Ok() {
+        Class expected = BalanceHandlerImpl.class;
+        Class actual = strategy.get(FruitTransaction.Operation.BALANCE).getClass();
+        assertEquals(expected,actual);
+    }
+
+    @Test
+    public void get_supplyTransactionHandler_Ok() {
+        Class expected = SupplyHandlerImpl.class;
+        Class actual = strategy.get(FruitTransaction.Operation.SUPPLY).getClass();
+        assertEquals(expected,actual);
```

Comment:
```suggestion
        assertEquals(expected, actual);
```
add spaces
not only here but in other places